+ source ../common/common_lib.sh
++ source /root/mugen-riscv/libs/locallibs/common_lib.sh
+++ python3 --version
+++ '[' 0 -eq 0 ']'
+++ source /root/mugen-riscv/libs/locallibs/common_lib_python.sh
+ main
++ type -t post_test
+ '[' -n function ']'
+ trap post_test EXIT INT HUP TERM
+ rpm -qa
+ grep expect
++ type -t config_params
+ '[' -n '' ']'
++ type -t pre_test
+ '[' -n function ']'
+ pre_test
+ LOG_INFO 'Start environmental preparation.'
+ message='Start environmental preparation.'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level info --message 'Start environmental preparation.'
Tue Feb 14 00:48:21 2023 - INFO  - Start environmental preparation.
+ DNF_INSTALL arptables
+ pkgs=arptables
+ node=1
+ '[' -z '' ']'
+ tmpfile=
++ python3 /root/mugen-riscv/libs/locallibs/rpm_manage.py install --pkgs arptables --node 1 --tempfile ''
+ tmpfile2=/tmp/tmpt4glndpx
+ '[' -z '' ']'
+ tmpfile=/tmp/tmpt4glndpx
+ LOG_INFO 'End of environmental preparation!'
+ message='End of environmental preparation!'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level info --message 'End of environmental preparation!'
Tue Feb 14 00:49:40 2023 - INFO  - End of environmental preparation!
++ type -t run_test
+ '[' -n function ']'
+ run_test
+ LOG_INFO 'Start testing...'
+ message='Start testing...'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level info --message 'Start testing...'
Tue Feb 14 00:49:41 2023 - INFO  - Start testing...
+ test_execution arptables.service
+ service=arptables.service
++ date '+%Y-%m-%d %T'
+ log_time='2023-02-14 00:49:42'
+ test_restart arptables.service
+ service=arptables.service
+ systemctl restart arptables.service
Job for arptables.service failed because the control process exited with error code.
See "systemctl status arptables.service" and "journalctl -xeu arptables.service" for details.
+ CHECK_RESULT 1 0 0 'arptables.service restart failed'
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log='arptables.service restart failed'
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n 'arptables.service restart failed'
+ LOG_ERROR 'arptables.service restart failed'
+ message='arptables.service restart failed'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'arptables.service restart failed'
Tue Feb 14 00:49:44 2023 - ERROR - arptables.service restart failed
+ (( exec_result++ ))
+ LOG_ERROR '../common/common_lib.sh line 34'
+ message='../common/common_lib.sh line 34'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message '../common/common_lib.sh line 34'
Tue Feb 14 00:49:45 2023 - ERROR - ../common/common_lib.sh line 34
+ return 0
+ SLEEP_WAIT 5
+ wait_time=5
+ cmd=
+ mode=1
+ python3 /root/mugen-riscv/libs/locallibs/sleep_wait.py --time 5 --cmd '' --mode 1
+ grep 'Active: active'
+ systemctl status arptables.service
+ CHECK_RESULT 1 0 0 'arptables.service restart failed'
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log='arptables.service restart failed'
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n 'arptables.service restart failed'
+ LOG_ERROR 'arptables.service restart failed'
+ message='arptables.service restart failed'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'arptables.service restart failed'
Tue Feb 14 00:49:54 2023 - ERROR - arptables.service restart failed
+ (( exec_result++ ))
+ LOG_ERROR '../common/common_lib.sh line 37'
+ message='../common/common_lib.sh line 37'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message '../common/common_lib.sh line 37'
Tue Feb 14 00:49:56 2023 - ERROR - ../common/common_lib.sh line 37
+ return 0
+ systemctl stop arptables.service
+ CHECK_RESULT 0 0 0 'arptables.service stop failed'
+ actual_result=0
+ expect_result=0
+ mode=0
+ error_log='arptables.service stop failed'
+ '[' -z 0 ']'
+ '[' 0 -eq 0 ']'
+ test 0x '!=' 0x
+ return 0
+ SLEEP_WAIT 5
+ wait_time=5
+ cmd=
+ mode=1
+ python3 /root/mugen-riscv/libs/locallibs/sleep_wait.py --time 5 --cmd '' --mode 1
+ grep 'Active: inactive'
+ systemctl status arptables.service
+ CHECK_RESULT 1 0 0 'arptables.service stop failed'
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log='arptables.service stop failed'
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n 'arptables.service stop failed'
+ LOG_ERROR 'arptables.service stop failed'
+ message='arptables.service stop failed'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'arptables.service stop failed'
Tue Feb 14 00:50:05 2023 - ERROR - arptables.service stop failed
+ (( exec_result++ ))
+ LOG_ERROR '../common/common_lib.sh line 42'
+ message='../common/common_lib.sh line 42'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message '../common/common_lib.sh line 42'
Tue Feb 14 00:50:07 2023 - ERROR - ../common/common_lib.sh line 42
+ return 0
+ systemctl start arptables.service
Job for arptables.service failed because the control process exited with error code.
See "systemctl status arptables.service" and "journalctl -xeu arptables.service" for details.
+ CHECK_RESULT 1 0 0 'arptables.service start failed'
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log='arptables.service start failed'
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n 'arptables.service start failed'
+ LOG_ERROR 'arptables.service start failed'
+ message='arptables.service start failed'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'arptables.service start failed'
Tue Feb 14 00:50:10 2023 - ERROR - arptables.service start failed
+ (( exec_result++ ))
+ LOG_ERROR '../common/common_lib.sh line 44'
+ message='../common/common_lib.sh line 44'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message '../common/common_lib.sh line 44'
Tue Feb 14 00:50:11 2023 - ERROR - ../common/common_lib.sh line 44
+ return 0
+ SLEEP_WAIT 5
+ wait_time=5
+ cmd=
+ mode=1
+ python3 /root/mugen-riscv/libs/locallibs/sleep_wait.py --time 5 --cmd '' --mode 1
+ systemctl status arptables.service
+ grep 'Active: active'
+ CHECK_RESULT 1 0 0 'arptables.service start failed'
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log='arptables.service start failed'
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n 'arptables.service start failed'
+ LOG_ERROR 'arptables.service start failed'
+ message='arptables.service start failed'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'arptables.service start failed'
Tue Feb 14 00:50:25 2023 - ERROR - arptables.service start failed
+ (( exec_result++ ))
+ LOG_ERROR '../common/common_lib.sh line 47'
+ message='../common/common_lib.sh line 47'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message '../common/common_lib.sh line 47'
Tue Feb 14 00:50:27 2023 - ERROR - ../common/common_lib.sh line 47
+ return 0
+ test_enabled arptables.service
+ service=arptables.service
++ systemctl is-enabled arptables.service
+ state=disabled
+ '[' disabled == enabled ']'
+ '[' disabled == disabled ']'
++ systemctl enable arptables.service
++ grep 'Created symlink'
++ awk '{print $3}'
+ symlink_file=/etc/systemd/system/multi-user.target.wants/arptables.service
+ find /etc/systemd/system/multi-user.target.wants/arptables.service
/etc/systemd/system/multi-user.target.wants/arptables.service
+ CHECK_RESULT 0 0 0 'arptables.service enable failed'
+ actual_result=0
+ expect_result=0
+ mode=0
+ error_log='arptables.service enable failed'
+ '[' -z 0 ']'
+ '[' 0 -eq 0 ']'
+ test 0x '!=' 0x
+ return 0
+ systemctl disable arptables.service
Removed /etc/systemd/system/multi-user.target.wants/arptables.service.
+ find /etc/systemd/system/multi-user.target.wants/arptables.service
find: ‘/etc/systemd/system/multi-user.target.wants/arptables.service’: No such file or directory
+ CHECK_RESULT 1 0 1 'arptables.service disable failed'
+ actual_result=1
+ expect_result=0
+ mode=1
+ error_log='arptables.service disable failed'
+ '[' -z 1 ']'
+ '[' 1 -eq 0 ']'
+ test 1x == 0x
+ return 0
+ grep -v -i 'DEBUG\|INFO\|WARNING'
+ journalctl --since '2023-02-14 00:49:42' -u arptables.service
+ grep -i 'fail\|error'
Feb 14 00:49:42 openEuler-riscv64 systemd[1]: arptables.service: Main process exited, code=exited, status=1/FAILURE
Feb 14 00:49:42 openEuler-riscv64 systemd[1]: arptables.service: Failed with result 'exit-code'.
Feb 14 00:49:42 openEuler-riscv64 systemd[1]: Failed to start Automates a packet filtering firewall with arptables.
Feb 14 00:50:08 openEuler-riscv64 systemd[1]: arptables.service: Main process exited, code=exited, status=1/FAILURE
Feb 14 00:50:08 openEuler-riscv64 systemd[1]: arptables.service: Failed with result 'exit-code'.
Feb 14 00:50:08 openEuler-riscv64 systemd[1]: Failed to start Automates a packet filtering firewall with arptables.
+ CHECK_RESULT 0 0 1 'There is an error message for the log of arptables.service'
+ actual_result=0
+ expect_result=0
+ mode=1
+ error_log='There is an error message for the log of arptables.service'
+ '[' -z 0 ']'
+ '[' 1 -eq 0 ']'
+ test 0x == 0x
+ test -n 'There is an error message for the log of arptables.service'
+ LOG_ERROR 'There is an error message for the log of arptables.service'
+ message='There is an error message for the log of arptables.service'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'There is an error message for the log of arptables.service'
Tue Feb 14 00:50:54 2023 - ERROR - There is an error message for the log of arptables.service
+ (( exec_result++ ))
+ LOG_ERROR '../common/common_lib.sh line 28'
+ message='../common/common_lib.sh line 28'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message '../common/common_lib.sh line 28'
Tue Feb 14 00:50:56 2023 - ERROR - ../common/common_lib.sh line 28
+ return 0
+ test_reload arptables.service
+ service=arptables.service
+ systemctl start arptables.service
Job for arptables.service failed because the control process exited with error code.
See "systemctl status arptables.service" and "journalctl -xeu arptables.service" for details.
+ systemctl reload arptables.service
+ grep 'Job type reload is not applicable'
Failed to reload arptables.service: Job type reload is not applicable for unit arptables.service.
+ CHECK_RESULT 0 0 0 'Job type reload is not applicable for unit arptables.service'
+ actual_result=0
+ expect_result=0
+ mode=0
+ error_log='Job type reload is not applicable for unit arptables.service'
+ '[' -z 0 ']'
+ '[' 0 -eq 0 ']'
+ test 0x '!=' 0x
+ return 0
+ systemctl status arptables.service
+ grep 'Active: active'
+ grep 'inactive (dead)'
+ systemctl status arptables.service
+ return 1
+ LOG_INFO 'Finish test!'
+ message='Finish test!'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level info --message 'Finish test!'
Tue Feb 14 00:51:00 2023 - INFO  - Finish test!
+ CASE_RESULT 0
+ case_re=0
+ test -z 6
+ test 6 -gt 0
+ LOG_ERROR 'failed to execute the case.'
+ message='failed to execute the case.'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'failed to execute the case.'
Tue Feb 14 00:51:01 2023 - ERROR - failed to execute the case.
+ exit 6
+ post_test
+ LOG_INFO 'start environment cleanup.'
+ message='start environment cleanup.'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level info --message 'start environment cleanup.'
Tue Feb 14 00:51:03 2023 - INFO  - start environment cleanup.
+ DNF_REMOVE
+ node=1
+ pkg_list=
+ mode=0
+ [[ -z /tmp/tmpt4glndpx ]]
+ '[' 0 -ne 0 ']'
+ '[' 1 == 0 ']'
+ python3 /root/mugen-riscv/libs/locallibs/rpm_manage.py remove --node 1 --pkgs '' --tempfile /tmp/tmpt4glndpx
+ '[' 0 -ne 0 ']'
+ LOG_INFO 'Finish environment cleanup!'
+ message='Finish environment cleanup!'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level info --message 'Finish environment cleanup!'
Tue Feb 14 00:51:40 2023 - INFO  - Finish environment cleanup!
